src/Microsoft.Data.Analysis.nuget/Microsoft.Data.Analysis.nuget.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you upload the resulting `.nupkg` from calling `dotnet pack` on this project? I want to take a look at it to see what it all contains.

Reply:
I'm not sure where do you want it uploaded. Anyway, I will try to upload it after the new project is in the right place (after changes are accepted).

Reply:
You can rename it as a `.zip` file and upload it to GitHub:

![image](https://user-images.githubusercontent.com/8291187/85162030-46876080-b226-11ea-9c4b-450333e08260.png)

Reply:
When trying to upload, I get this:

![Untitled](https://user-images.githubusercontent.com/15055082/85163214-d1864c00-b262-11ea-8306-dfcd3e66ae17.png)